src/dbSta/include/db_sta/dbNetwork.hh

```diff
@@ -240,6 +243,10 @@ class dbNetwork : public ConcreteNetwork
   double dbuToMeters(int dist) const;
   int metersToDbu(double dist) const;
 
+  // hierarchy handler
+  void setHierarchy() { hierarchy_ = true; }
+  bool isHierarchical() const { return hierarchy_ == true; }
```

Comment:
Why have this public method given you comment "// private to dbNetwork, never used by anyone else outside of dbNetwork."?